### ISSUE TITLE
Subscribing to blocklist doesn't result in blocking the accounts

### ISSUE THREAD

**Issue by dtiersch** [jsha/blocktogether#212]
A few days ago I subscribed to a rather big (>20.000) blocklist, but when I checked on a handful of accounts on the list, none have been blocked.
Is there any debug data I could provide to help resolving the problem?

**Comment by jsha** [comment]
Block Together wasn't working for the last day or so due to a disk full
event. I just restored it a couple of hours ago, so your blocks should be
in progress now.

**Comment by dtiersch** [comment]
I just checked again for a few accounts (on the first and the last page if that does matter) but every checked account still shows as not blocked

**Comment by jsha** [comment]
What's your screen name? Can you try unsubscribing and resubscribing?

**Comment by dtiersch** [comment]
My screen name is `@Salandas`. I did try resubscribing but I got a gateway error; the list appears afterwards in the subscriptions screen, though.

**Comment by dtiersch** [comment]
That's strange: When I now open the address I got, I get the message `Error: No such block list.`. But it is still listed under subscriptions...

**Comment by jsha** [comment]
Thanks for the info. I think I've found the problem and fixed it. Try
unsubscribing and resubscribing once more.

**Comment by dtiersch** [comment]
This seemed to work. The accounts are now listed as pending on my Actions page and the first ones are already done, so I guess all is fine now. Thanks
